helpers/inventory.js

```diff
@@ -85,7 +85,8 @@ module.exports.updateLimits = function(frozen, room, refrigerated) {
         var error = new Error('Can\'t set refrigerated limit to under current capacity');
         throw error;
       }
-      resolve(inv.save());
```

Comment:
the old version will also return the ingredient when resolved, except now if inv.save() fails we'll have a dreaded unhandledpromiserejection